### ISSUE TITLE
graphconv with sequence data

### ISSUE THREAD

**Issue by cswwp** [danielegrattarola/spektral#98]
`    X_in = tf.keras.layers.Input(shape=(107, 17,))
    print('X_in:', X_in.shape)
    A_in = tf.keras.layers.Input((107, 107,), sparse=True)  
    print('A_in:', A_in.shape)

    X_1 = spektral.layers.GraphConv(16, 'relu')([X_in, A_in])
    print('X_1:', X_1.shape)

    X_1 = tf.keras.layers.Dropout(0.5)(X_1)
    print('X_1:', X_1.shape)

    X_2 = spektral.layers.GraphConv(5, 'softmax')([X_1, A_in])
    print('X_2:', X_2.shape)`

i want use gc with sequence data, my input X_in shape is [batch, step, feature_dim], and adjust matrix A_in shape is batch, 107, 107, and then i use     X_1 = spektral.layers.GraphConv(16, 'relu')([X_in, A_in]),   but the X_1 shape is [None, None, 16], not [None, 107, 6], i'm confused, anyone could help me?  i check the doc, it show output should just change the last dim to graphconv's channel, and other dim keep same with input shape:
![image](https://user-images.githubusercontent.com/11901683/94047690-05927f80-fe05-11ea-8a56-6fbab1ad84ee.png)

output info 
![image](https://user-images.githubusercontent.com/11901683/94047758-20fd8a80-fe05-11ea-9141-5cfd651ce828.png)

**Comment by cswwp** [comment]
Hi @danielegrattarola. Do you know how to fix it?

**Comment by StefanBloemheuvel** [comment]
Could you tell something about the sequence data you mentioned, was it time series data? And did you succeed with fixing your problem now?
